temporian/implementation/numpy/data/event.py

```diff
@@ -135,16 +202,12 @@ def from_dataframe(
             ... )
             >>> event = NumpyEvent.from_dataframe(df, index_names=["product_id"])
         """
-        if index_names is None:
```

Comment:
Actually, we have to leave the None in there. If we have a mutable object as a default value, then that object reference will be shared between different calls of the function and will end up with unexpected results. 

[https://web.archive.org/web/20201002220217/http://effbot.org/zone/default-values.htm ](https://web.archive.org/web/20201002220217/http://effbot.org/zone/default-values.htm )